runtime/vm/createramclass.cpp

```diff
@@ -1995,6 +1995,13 @@ internalCreateRAMClassDone(J9VMThread *vmThread, J9ClassLoader *classLoader, J9R
 			}
 		}
 
+		/* link classes in segment */
+		if (NULL != segment) {
+			/* segment is NULL in failure paths */
```

Comment:
I would move this comment (merge it with the one above) since this is in the not NULL path.